src/components/VerificationCode/VerificationCode.css.js

```diff
@@ -0,0 +1,119 @@
+import styled from '../styled'
+import { getColor } from '../../styles/utilities/color'
+import Icon from '../Icon'
+import Tooltip from '../Tooltip'
+
+export const VerificationCodeFieldUI = styled('div')`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 500px;
+  height: 74px;
+  border: 1px solid #c5ced6;
+  border-radius: 3px;
+  background-color: #fff;
+
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:focus-within {
```

Comment:
this is awesome! i did not know about `focus-within`! It does look like this is not supported by IE though. Should make sure the component still looks/functions acceptably on IE.

Reply:
This is to be used in HS App if I'm not mistaken? We don't officially support IE there (https://docs.helpscout.com/article/1292-supported-browsers-and-system-requirements) only latest Edge, which happens to be the brand new chromium based one 🕺

And I would consider this as a "graceful degradation", so IE users still get an acceptable experience, but of course if there's a specific requirement for this component, pls let me know!

Reply:
ok sounds good. I'm fine with it if it works on latest edge!